nss-day-cohort-30/bangazon-api-ambitious-albatrosses
# Stephen

Fixes:



Changes proposed in this request & reasons behind organizational or architectural decisions:

Implemented controllers for Training Programs and Computers


Steps to test:

Insert dummy data into database
Run all tests. 
-
-
-

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-
Commit message:
-
Added controllers and tests for Training Programs and Computers

Link to feature ticket:
-

[Training Programs](https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/issues/8)
[Computers](https://github.com/nss-day-cohort-30/bangazon-api-ambitious-albatrosses/issues/9)
@ambitions-albatrosses
